### PR TITLE
docs: Updates Readme with Description

### PR DESCRIPTION
**Pull request by christopheralex**
In addition to 
- [x] Contributors 
- [x] Proposed Design Layout
- [x] License 
- [x] Contributing

### CODE REVIEW

**Comment by christopheralex** [comment]
Please do let me know if you guys think there has to be more content in the description. 
I have kept it as abstract as possible as Florencia mentions not to get technical and it has to be to general audience.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+
+## Dashboard description
+
+The app contains a landing page that shows 4 graphs and a single tool box for controlling the graphs individually and together.
+
+- Top artists with an average popularity score filtered by Genre 
+    * Here the user can select a particular `genre` of music by using the drop down list for genre located on the left hand side of the dashboard. 
+    * The plot shows the artist and their corresponding popularity score for the specific genre.
+- Popularity Trend of Specific Artists across time
+    * The user on choosing a specific artist from a different down drop list, will be provided with a line chart of the artist's avergae popularity score over the years.
```
Comment (jennifer-hoang) [inline README.md:13]:
```suggestion
    * The user on choosing a specific artist from a different down drop list, will be provided with a line chart of the artist's average popularity score over the years.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,40 @@
 # spotify-explorer-py
 
-Authors - Christopher Alexander , Jennifer Hoang, Thea Wenxin, Michelle 
+Spotify explorer is a dashboard implemented in Dash in order to help business analysts, product content managers visualize and understand the current trends in the music industry through the spotify dataset.
```
Comment (jennifer-hoang) [inline README.md:3]:
```suggestion
Spotify Explorer is a dashboard implemented in Dash in order to help business analysts, product content managers visualize and understand the current trends in the music industry through the Spotify dataset. 
```

**Review by jennifer-hoang — CHANGES_REQUESTED** [review]
Looks good to me Chris! I've added some minor typo edits. 

Thanks!

**Comment by christopheralex** [comment]
> Looks good to me Chris! I've added some minor typo edits.
> 
> Thanks!

Committed all new changes.

**Review by jennifer-hoang — APPROVED** [review]

**Review by xiangwxt — COMMENTED** [review]
Hi Chris, the readme file is really conclusive of what's in our proposal, nice done. The only thing I would suggest is maybe you want to add a link to code of conduct to it. All others look really good.

**Pull request merged**